### PR TITLE
INT - B-21412 & B-21413 Prime sim - create international crate

### PR DESCRIPTION
**Pull request by taeJungCaci**
## Summary

This adds the ability for Prime user to create International Crate/Uncrate (ICRT & ICURT) service items similar to Domestic Crate/Uncrate. ICRT includes 2 boolean checkboxes (standalone and external crate) that ICURT does not have. `Market` field with possible values `OCONUS` and `CONUS` has been added to `MTOServiceItemInternationalCrating.yaml` which is derived from corresponding `address.isOconus`.

Also the service item `ICRTSA` has been removed from the DB and code.

- FF under `enable_alaska`.
- Added International Crate and International Uncrate as options when adding service item in Prime Sim.
- `External Crate` boolean checkbox added to International Crating.
- `Market` field added to `MTOServiceItemInternationalCrating.yaml` with enum values: `CONUS` or  `OCONUS`.
- ICRTSA service item has been removed from DB tables (`service_params, re_intl_accessorial_prices, and re_services).
- `external_crate` column added to `mto_service_items` table.

### How to test

1. In your `.envrc` file -> change to `export FEATURE_FLAG_ENABLE_ALASKA=true` to turn on Alaska FF locally.
2. Close your server -> run `direnv allow` and then restart your server.
3. Create a move and get the the step where the Prime can see the move and is able to create a service item (or use `HHGMoveInSITEndsTomorrow` testharness.
4. In Prime Sim -> find the move.
5. Click `Add Service Item` button for the shipment.
6. Select `International Crating` for "Service item type".
7. Select `International Crating` for "Service item code".
8. You should see `Standalone Crate` and `External Crate` options for "International crating" service item code and not "International Uncrating".
9. Test all the options to verify that you are able to create those service items.
10. Make sure item length/width/height is less than the Crate length/width/height.
11. To check if the service item was created -> find the move as TOO and go to move task order tab -> should see it under requested service items table (details will be empty).

## Screenshots
Screenshot 2024-11-18 at 11 05 57 AM
Screenshot 2024-11-18 at 11 06 28 AM
Screenshot 2024-11-18 at 12 37 21 PM
Screenshot 2024-11-18 at 12 37 27 PM
image

### CODE REVIEW

**Review by KonstanceH — APPROVED** [review]
Tried different combinations in testing and static look at code looks good to me. 
LGTM!

**Review by brooklyn-welsh — APPROVED** [review]
Tested and working. Code LGTM ✅

File: migrations/app/schema/20241111203514_add_external_crate_and_remove_icrtsa.up.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+-- Add external_crate to mto_service_items
+ALTER TABLE mto_service_items ADD COLUMN IF NOT EXISTS external_crate bool NULL;
```
Comment (deandreJones) [inline migrations/app/schema/20241111203514_add_external_crate_and_remove_icrtsa.up.sql:2]: any reason why this isn't default false

Comment (taeJungCaci) [reply migrations/app/schema/20241111203514_add_external_crate_and_remove_icrtsa.up.sql:2]: not particularly - just copied how others were done. Thought maybe they also want a null state for these booleans to account for not being used in these service items.

**Review by deandreJones — APPROVED** [review]

**Pull request merged**